.env

```diff
@@ -0,0 +1 @@
+DCS_DZ_TYPEORM_LOGGER='simple-console'
```

Comment:
[question] is this supposed to be commited?

Reply:
Just binned that :)